ethereum/core/build.gradle

```diff
@@ -15,6 +15,10 @@
 
 apply plugin: 'java-library'
 
+repositories {
+  maven { url "https://artifacts.consensys.net/public/maven/maven/" }
+}
+
```

Comment:
This should not be needed here and should be removed.  The repository is already enumerated in the parent build file.